package.json

```diff
@@ -18,6 +18,7 @@
     "react-map-gl": "^5.2.3",
     "react-redux": "^7.1.3",
     "react-responsive-carousel": "^3.2.4",
+    "react-router": "^5.1.2",
```

Comment:
delete it

Reply:
I should delete line 21 :  react-router wala, right?

And then npm i the same again

Reply:
yes